ESP32LapTimer/src/RX5808.cpp

```diff
@@ -41,49 +41,25 @@ void rxWrite(uint8_t addressBits, uint32_t dataBits, uint8_t CSpin) {
 }
 
 void rxWriteNode(uint8_t node, uint8_t addressBits, uint32_t dataBits) {
-  switch (node) {
-    case 0:
-      rxWrite(addressBits, dataBits, CS1);
-      break;
-    case 1:
-      rxWrite(addressBits, dataBits, CS2);
-      break;
-    case 2:
-      rxWrite(addressBits, dataBits, CS3);
-      break;
-    case 3:
-      rxWrite(addressBits, dataBits, CS4);
-      break;
-    case 4:
-      rxWrite(addressBits, dataBits, CS5);
-      break;
-    case 5:
-      rxWrite(addressBits, dataBits, CS6);
-      break;
-  }
+  rxWrite(addressBits, dataBits, CS_PINS[node]);
```

Comment:
Same here. In this case we might just do nothing?

Reply:
If we're doing it for the ADC we might as well do it here?

Reply:
This part is a little bit different since
1. This is the old behavior ;)
2. If we specify a wrong ADC we get one bad sample. If we specify a wrong node here, we may set a wrong setting

If we want a consistent behavior for both cases we should skip them both

Reply:
I see your point. Ill add the check to the ADC and skip it here.

Reply:
Sorry, I meant do nothing like do nothing if "node" is invalid:
```
if (node < MAX_NUM_RECEIVERS) {
  rxWrite(addressBits, dataBits, CS_PINS[node]);
}
```

Reply:
Ah :D

Reply:
Addressed in 17a5add